dockstore-webservice/src/main/java/io/dockstore/webservice/core/AppTool.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *    Copyright 2019 OICR
```

Comment:
I updated the Dockstore wiki primer to have what I think is the correct copyright, you can use that